public/data/khushisaxena.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Khushi Saxena",
+    "bio": "Software Developer || Coder || Open Source Enthusiast",
+    "avatar": "https://github.com/Khushi0321.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/Khushi0321",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/khushi_321",
+        "icon": "twitter"
+      },
+      {
+        "name": "Check my Portfolio",
+        "url": "https://khushi0321.github.io/portfolio/#/",
+        "icon": "link"
+      }
+    ]
+  }
+  
```

Comment:
```suggestion

```

Reply:
Mistakenly, I had deleted the forked repository, Now I am not able to make these changes.